From 7ac9c657ab2bacb1e7049703dce563143cb88b35 Mon Sep 17 00:00:00 2001
From: Sergei Miroshnichenko <s.miroshnichenko@yadro.com>
Date: Fri, 18 Dec 2020 16:47:27 +0300
Subject: [PATCH 22/57] PCI: hotplug: Enable the movable BARs feature by
 default

This is the last patch in the series which implements the essentials of the
movable BARs feature, so it is turned on by default now.

Tested on x86_64; and with extra patches applied, it also works on ppc64le
(PowerNV).

In case of problems it is still can be overridden by the following command
line option:

  pcie_movable_bars=off

Signed-off-by: Sergei Miroshnichenko <s.miroshnichenko@yadro.com>
---
 drivers/pci/pci.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pci/pci.c b/drivers/pci/pci.c
index 111111111..111111111 100644
--- a/drivers/pci/pci.c
+++ b/drivers/pci/pci.c
@@ -82,7 +82,7 @@ bool pci_reset_supported(struct pci_dev *dev)
 int pci_domains_supported = 1;
 #endif
 
-bool pci_can_move_bars;
+bool pci_can_move_bars = true;
 
 #define DEFAULT_CARDBUS_IO_SIZE		(256)
 #define DEFAULT_CARDBUS_MEM_SIZE	(64*1024*1024)

